app/views/users/choose_identity.html.erb

```diff
@@ -7,7 +7,7 @@
 	<%- if @current_user.is_a?(PubcookieUser) -%>
 		<li><%= link_to "Dream Project mentor", update_identity_path(:identity => "Mentor"), :method => :post %></li>
 	<%- else -%>
-		<li><%= link_to "high-school student", update_identity_path(:identity => "Student"), :method => :post %></li>
+		<li><%= link_to "high-school student", update_identity_path(:identity => "Participant"), :method => :post %></li>
```

Comment:
Gotcha! The intake form currently is for a `Participant`. So how would we change a `Student` to a `Participant`? Because if they select "high school student", the intake form would render an error for `nil`.